data/language/en-US.txt

```diff
@@ -883,7 +883,7 @@ STR_DTLS    :You are a successful business tycoon on long sabbatical who desires
 <South America - Inca Lost City>
 STR_SCNR    :South America - Inca Lost City
 STR_PARK    :Lost City Founder
-STR_DTLS    :To further boost local tourism you must construct a park that is in tune with its surroundings, and has height restrictions.
+STR_DTLS    :To further boost local tourism you must construct a park that is in tune with its surroundings.
```

Comment:
Only en-GB is maintained in this repository, the rest is maintained on Localisation. You’ll need to revert this file.

Reply:
Gotcha; thanks!